[Phase 1] Fix&update _race_interstitial modal on dashboard page

levadadenys opened the pull request:
<!--
  A summary of the change, including any relevant background, motivation, and context.
  If relevant, include a description, screenshots, and/or video of the existing and new behavior.
-->
### [Phase 1] Fix&update _race_interstitial modal on dashboard page

Before:
![Screenshot from 2023-01-30 15-07-47](https://user-images.githubusercontent.com/22244040/215486088-e680f7be-6714-4bfe-8436-14c974a35fb5.png)

After:
![Screenshot from 2023-01-30 22-02-12](https://user-images.githubusercontent.com/22244040/215583262-d95903d1-dd98-488f-8c55-ef34d7c1d6ed.png)


## Links

<!--
  Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
-->

<!--
- spec: []()
- jira ticket: []()
-->

## Testing story

<!--
  Does your change include appropriate tests?
  If so, please describe how the tests included in this PR are sufficient.
  If not, please explain why this change does not need to be tested.
-->

<!-- Other aspects to consider. Delete any sections that are not relevant to your change. -->

## Deployment strategy

## Follow-up work

<!--
  List (ideally with Jira links) any clean-up or technical debt that will be addressed in future work.
-->

## Privacy

<!--
  1.	Does this change involve the collection, use, or sharing of new Personal Data?
  2.	Does this change involve a new or changed use or sharing of existing Personal Data?
-->

## Security

<!-- Link to Jira task(s) where sensitive security issues are discussed privately. -->

## Caching

## PR Checklist:

<!--
  The final step! Before you create your PR, double-check that everything is in order.
  Change [ ] to [X] during creation to check boxes.
-->

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

breville commented:
Could you add a before screenshot to illustrate the fix?

Also, the screenshot still has a couple possible issues: 
- it seems odd that the body text ("Please tell us...") is smaller than the text for the options.  Normally they'd be the same size?
- it also seems odd that the heading ("Optional: ...") is smaller than the text for the options.

levadadenys commented:
> Could you add a before screenshot to illustrate the fix?
> 
> Also, the screenshot still has a couple possible issues:
> 
> * it seems odd that the body text ("Please tell us...") is smaller than the text for the options.  Normally they'd be the same size?
> * it also seems odd that the heading ("Optional: ...") is smaller than the text for the options.

Thanks for the hints! Fixed and updated the PR

breville commented on dashboard/app/views/layouts/_race_interstitial.html.haml:
```diff
@@ -1,9 +1,9 @@
   .modal-dialog
     .modal-content.no-modal-icon
-      %h2.custom-h2= t('race_interstitial.title')
+      %h2.custom-h2{style: "margin-top: 20px; margin-bottom: 10px; font-size: 1.7em;"}= t('race_interstitial.title')
```
Do we have a policy on what should be inline styling versus in inline CSS or external SCSS?

levadadenys replied:
Good point, thank you. I've moved this styles to interstitial.scss

breville reviewed (approved):
Styling improvements look good.
